rails/lib/materials/data_helpers.rb

```diff
@@ -442,8 +442,8 @@ def links_for_material( material,
       if current_visitor.has_role?('admin') && material.respond_to?(:materials_collections)
         links[:assign_collection] = {
           text: "Add to Collection",
-          url: "javascript:void(0)",
-          onclick: "get_Assign_To_Collection_Popup(#{material.id},'#{material.class.to_s}','#{t('material').pluralize.capitalize}',#{skip_lightbox_reloads})"
+          target: "_blank",
+          url: "/search/get_current_material_unassigned_collections/?material_id=#{material.id}&material_type=#{material.class.to_s}"
```

Comment:
Same thing here: if this was moved to the ExternalActivity controller then a route path helper can be used here:
`edit_collections_external_activity_path(@material)`